### ISSUE TITLE
[Book 3] Chapter 1: Incorrect reference to "asset" section

### ISSUE THREAD

**Issue by rupsis** [RayTracing/raytracing.github.io#1371]
In Book 3, the introduction states: 
```
These books have been formatted to print well directly from your browser. We also include PDFs of each book?, in the “Assets” section. 
```

1) incorrect `?`
2) As far as I'm aware, there isn't an "assets" section? 

Should be updated to:
```
These books have been formatted to print well directly from your browser. 
```

(and if there's a hosted source of the PDF's, we should link to that.)

Happy to make the changes.

**Comment by hollasch** [comment]
If you look at book 1 source, you'll see that this is a link to `[releases]`, which should be defined with other links near the bottom of the file. It's present in book 1, but lacking for books 2 and 3.

You can see what this is pointing to by looking at the Assets section in the [release pages for v4.0.0-alpha](https://github.com/RayTracing/raytracing.github.io/releases/tag/v4.0.0-alpha.1) and [v3.2.3](https://github.com/RayTracing/raytracing.github.io/releases/tag/v3.2.3).

Please add the missing link definitions for books 2 and 3.

For extra credit, update all prior releases with the PDF files as far back as you care to go.

**Comment by rupsis** [comment]
Addressed in #1387
